lib/cocoa/src/protocol/TCompactProtocol.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#import "TProtocol.h"
+#import "TTransport.h"
+#import "TProtocolFactory.h"
+
+@interface TCompactProtocol : NSObject<TProtocol>
+
+-(id)initWithTransport:(id <TTransport>)transport;
```

Comment:
It would be nice to be consistent with the layout style used in the rest of the Obj-C library. Specifically, it uses a lot more whitespace in method decls. See for example [TProcessor.h](https://github.com/apache/thrift/blob/master/lib/cocoa/src/TProcessor.h#L24-L29) which looks like:

```objc
@protocol TProcessor <NSObject>

- (BOOL) processOnInputProtocol: (id <TProtocol>) inProtocol
                 outputProtocol: (id <TProtocol>) outProtocol;

@end
```

The main difference is adding space between the `-/+` leading the method decl/impl and the return type, between types and `*`s, and between the types and the selector chunks.

This affects all methods declared and implemented in this PR, but it should be an easy change with some regex magic :).